index.html

```diff
@@ -212,6 +212,39 @@ <h1 class="about-section-h1">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="form-section">
+            <h1 class="form-h1">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd like to<br>chat please get in touch.</h1>
+            <form action="https://formspree.io/f/xoqypwnp" method="post">
+                    <ul class="ul-for-form">
+                        <li>
+                            <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input type="email" name="user_email" id="mail" placeholder="Email Address" required>
+                        </li>
+                        <li class="txt"> 
+                            <textarea  class="text-input" name="user_message" id="msg" cols="10" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+                        </li>
+                        <li class="button-container">
+                            <input class="get-button" type="submit" name="submit" value="Get in touch">
+                        </li>
+                    </ul>
```

Comment:
Instead of using a list `ul` with `li` elements, you can remove them in order to use semantic HTML tags.

- [ ] [OPTIONAL] Remove the list `ul` with `li` elements in your form.